src/dashboard/src/components/charts/allocationTable/defaultData.ts

```diff
@@ -1,100 +1,103 @@
-export const tableRowData = [
+import { ITableRowData } from './ITableRowData';
+
+export const tableRowData: ITableRowData[] = [
   {
     server: 'Server Alpha',
     tenant: '',
     os: 'Ubuntu 20.04 LTS',
-    allocated: 10.0,
-    unused: 2.0,
+    capacity: 10.0,
```

Comment:
Keeping property names consistent throughout based on the data ServiceNow provides for us.